examples/basics/finetuning_vgg.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+'''
+Finetuning Example with vgg.tflearn. Using weights from VGG model to retrain
+network for a new task (your own dataset).All weights are restored except
+last layer (softmax) that will be retrained to match the new task (finetuning).
+'''
+import tflearn
+from tflearn.layers.estimator import regression
+import os
+
+
+def vgg16(placeholderX=None):
+    x = tflearn.input_data(shape=[None, 224, 224, 3], name='input',
+                           placeholder=placeholderX)
+
+    x = tflearn.conv_2d(x, 64, 3, activation='relu', scope='conv1_1')
+    x = tflearn.conv_2d(x, 64, 3, activation='relu', scope='conv1_2')
+    x = tflearn.max_pool_2d(x, 2, strides=2, name='maxpool1')
+
+    x = tflearn.conv_2d(x, 128, 3, activation='relu', scope='conv2_1')
+    x = tflearn.conv_2d(x, 128, 3, activation='relu', scope='conv2_2')
+    x = tflearn.max_pool_2d(x, 2, strides=2, name='maxpool2')
+
+    x = tflearn.conv_2d(x, 256, 3, activation='relu', scope='conv3_1')
+    x = tflearn.conv_2d(x, 256, 3, activation='relu', scope='conv3_2')
+    x = tflearn.conv_2d(x, 256, 3, activation='relu', scope='conv3_3')
+    x = tflearn.max_pool_2d(x, 2, strides=2, name='maxpool3')
+
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv4_1')
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv4_2')
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv4_3')
+    x = tflearn.max_pool_2d(x, 2, strides=2, name='maxpool4')
+
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv5_1')
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv5_2')
+    x = tflearn.conv_2d(x, 512, 3, activation='relu', scope='conv5_3')
+    x = tflearn.max_pool_2d(x, 2, strides=2, name='maxpool5')
+
+    x = tflearn.fully_connected(x, 4096, activation='relu', scope='fc6')
+    x = tflearn.dropout(x, 0.5, name='dropout1')
+
+    x = tflearn.fully_connected(x, 4096, activation='relu', scope='fc7')
+    x = tflearn.dropout(x, 0.5, name='dropout2')
+
+    x = tflearn.fully_connected(x, 12, activation='softmax', scope='fc8', restore=False)
```

Comment:
Here, can you change 12 by a variable? such as num_class, and add num_class as function argument, so that any user can easily adapt this example with its actual total number of class